kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkConnectorTest.java

```diff
@@ -38,5 +66,23 @@ public void connectorType() {
     Connector connector = new S3SinkConnector();
     assertTrue(SinkConnector.class.isAssignableFrom(connector.getClass()));
   }
+
+  @Test(expected = ConnectException.class)
```

Comment:
We don't want to get a ConnectException here. Instead, the `Config` returned from the `validate(...)` method should contain our message on the `s3.bucket.name` property.

What happens if the credentials are not correct? Do I get a "bucket does not exist" validation error? If so, should the message should be something like "The S3 bucket named '{}' could not be found. Check the S3 credentials and verify the '{}' bucket exists."

Reply:
Exactly. If the credentials are incorrect, we get a "bucket does not exist" error. So added the message you suggested.